packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -859,16 +885,23 @@
     }
 
     &__footer {
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+        bottom: 0;
+
         @include mobile-or-tablet-screen {
-            display: flex;
             flex-direction: row;
-            align-items: center;
-            justify-content: flex-end;
-            position: fixed;
-            bottom: 0;
+            width: 100%;
+            justify-content: center;
+        }
 
-            .proof-of-identity__submit-button {
-                width: 100%;
+        .proof-of-identity__submit-button {
+            @include mobile-or-tablet-screen {
+                width: 568px;
```

Comment:
Can we use `rem` instead of `px`
Thank you

Reply:
done 👍